README.md

```diff
@@ -145,3 +145,13 @@ Contributions are welcome & greatly appreciated, every little bit helps. Make co
 ```
 make test-docker
 ```
+
+### Interactive "Office Hours"
+
+The maintenance team plans to be available regularly for questions, comments,
+pings, etc for about an hour twice a week. The current schedule is:
+
+* 18:00 UTC (2:00pm US/Eastern) Mondays
```

Comment:
Can we mount it to DST instead of UTC? Good question is, what will we do, when Europe gets rid of DST.

Reply:
I can "pin it" to US/Eastern since that's my TZ and I'm proposing these :-).

Reply:
Done. When the time zones change and visitors get confused (& that would mean people are making use of these) we'll update as needed.